### PR TITLE
Update Grommet to 2.15.2

### PR DESCRIPTION
**Pull request by mcbouslog**
Package: app-content-pages, app-project, lib-classifier, lib-react-components

Describe your changes:
- updates Grommet to v2.15.2
- refactors tests, mainly to include shallow or mount wrapping component options

Note
- originally included grommet-icons update, thought that was working fine, it wasn't, will open related separate PR

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/Banners/components/Banner/Banner.js [hunk 1]
```diff
@@ -23,6 +23,20 @@ function Label () {
   )
 }
 
+export function Tooltip ({ mode, tooltipText }) {
```
Comment (mcbouslog) [inline packages/lib-classifier/src/components/Classifier/components/Banners/components/Banner/Banner.js:26]: should extract `Tooltip` (and related tests) to own file in `./components`?

Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/Banners/components/Banner/Banner.js:26]: Actually, we should replace it with the `Tooltip` that is part of the component library. This was written prior to that one I think.

Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/Banners/components/Banner/Banner.js:26]: This can be done in a separate PR.

**Review by srallen — COMMENTED** [review]
There was a style regression with one of the controls with the variable star LCV:

Screen Shot 2020-11-09 at 5 19 37 PM

This is not surprising because the styles I wrote for it are really hacky. I'll hold off giving approval and have a look at a fix for this tomorrow branching off of this one.

**Review by srallen — APPROVED** [review]
LGTM! Thanks!

**Comment by mcbouslog** [comment]
Closes #1591

**Pull request merged**